cherry_picker/cherry_picker/cherry_picker.py

```diff
@@ -381,11 +381,11 @@ def check_repo(self):
               help=("Path to config file, .cherry_picker.toml "
                     "from project root by default"),
               default=None)
-@click.argument('commit_sha1', 'The commit sha1 to be cherry-picked', nargs=1,
```

Comment:
Why remove the explanation? How is this related to the click dependency update?

Reply:
oh, they changed api..

Reply:
Explicitly forbade meaningless argument actually. The help was never displayed IIRC

Reply:
Oh, nevermind. I've already figured out that it fails with this arg under newer Click: https://travis-ci.org/python/core-workflow/jobs/491106736#L270